toolkit/themes/shared/design-system/design-tokens.json

```diff
@@ -55,6 +55,116 @@
       "value": "1px"
     }
   },
+  "box": {
+    "background": {
+      "color": {
+        "value": {
+          "light": "{color.white}",
+          "dark": "{color.gray.80}",
+          "prefersContrast": "{color.canvas}"
+        }
+      }
+    },
+    "shadow": {
+      "10": {
+        "value": "0 1px 4px {color.black.a10}"
```

Comment:
We may want / need a better way to represent shadow down the line (I doubt this would work for mobile platforms...). Seems out of scope for now but I can file a subtask/follow up at least.